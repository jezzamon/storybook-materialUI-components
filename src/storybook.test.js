import initStoryshots from '@storybook/addon-storyshots';
initStoryshots();
describe('groupByDay', () => {
  it('should aggregate events by day', () => {
    const events = [
      { createdAt: '2019-01-05T12:56:31.039Z', label: 'foo1' },
      { createdAt: '2019-01-05T09:12:43.456Z', label: 'foo2' },
      { createdAt: '2019-01-04T12:34:56.789Z', label: 'foo3' },
    ];
    expect(events).toEqual([
      { createdAt: '2019-01-05T12:56:31.039Z', label: 'foo1' },
      { createdAt: '2019-01-05T09:12:43.456Z', label: 'foo2' },
      { createdAt: '2019-01-04T12:34:56.789Z', label: 'foo3' },
    ]);
  });
});
