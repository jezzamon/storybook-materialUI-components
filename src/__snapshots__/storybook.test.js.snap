// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Avatar anonymous 1`] = `<div />`;

exports[`Storyshots Avatar basic 1`] = `<div />`;

exports[`Storyshots Avatar using fallback 1`] = `<div />`;

exports[`Storyshots Button another background 1`] = `<div />`;

exports[`Storyshots Button with background 1`] = `<div />`;

exports[`Storyshots EventItem anonymous 1`] = `<div />`;

exports[`Storyshots EventItem basic 1`] = `<div />`;

exports[`Storyshots EventItem long event name 1`] = `<div />`;

exports[`Storyshots EventList basic 1`] = `<div />`;

exports[`Storyshots EventList empty 1`] = `<div />`;

exports[`Storyshots Main Contained Buttons 1`] = `<div />`;

exports[`Storyshots Main Outlined Buttons 1`] = `<div />`;

exports[`Storyshots Main Text Buttons 1`] = `<div />`;

exports[`Storyshots TimelineLoaded basic 1`] = `<div />`;

exports[`Storyshots TimelineLoaded empty 1`] = `<div />`;

exports[`Storyshots TimelineLoaded fully loaded 1`] = `<div />`;

exports[`Storyshots Welcome To your new storbook : tada emoji 1`] = `<div />`;
